### PR TITLE
more fixes

### PR DESCRIPTION
**Pull request by mgxd**
* [x] ensure `invert_transform_flags` matches transforms
* [x] specify mindboggle positional arg `prov`

### CODE REVIEW

**Comment by mgxd** [comment]
we should also probably bump the version to `1.4.0`

File: mindboggle/mindboggle123 [hunk 1]
```diff
@@ -329,7 +329,7 @@ else:
         transformer_nn = Node(ApplyTransforms(), name="transformer_nn")
         transformer_nn.inputs.reference_image = IMAGE
         transformer_nn.inputs.dimension = 3
-        transformer_nn.inputs.invert_transform_flags = [False, False]
+        transformer_nn.inputs.invert_transform_flags = [False] * N
```
Comment (satra) [inline mindboggle/mindboggle123:332]: this should not be the case. corticalthickness generates two transform files that go through the merge node. this change should be reverted.

Comment (satra) [reply mindboggle/mindboggle123:332]: `N` on the other hand is the total number of atlases, which has no bearing on the quick option.

Comment (satra) [reply mindboggle/mindboggle123:332]: this change should be in the fusion section, and invert_transform should be a single element vector with `False`

**Comment by satra** [comment]
@mgxd - updated my branch

**Pull request closed without merging**